CMakeLists.txt

```diff
@@ -13,10 +13,12 @@ project(fastfetch
 
 if("${CMAKE_SYSTEM_NAME}" MATCHES ".*[Ll]inux.*")
     set(LINUX TRUE CACHE BOOL "..." FORCE) # LINUX means GNU/Linux, not just the kernel
+elseif("${CMAKE_SYSTEM_NAME}" MATCHES "MSYS|Windows")
+    set(WIN_MSYS TRUE CACHE BOOL "..." FORCE) # Windows on msys2
```

Comment:
Cmake enables a `WIN32` variable on Windows by default, can't we used that one?

If this patch doesn't work without MSys2, cmake also enables a `MSYS` variable there.

See https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Write-Platform-Checks#platform-variables